Bottom: 7b527f711768166a0eaa950c02710df6fdd564f1
Top:    8d8a4151ae69abe3400e70be16efbd8fe0e39eaf
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-20 11:57:53 -0400

Refresh of efika-audio

---

diff --git a/sound/soc/soc-core.c b/sound/soc/soc-core.c
index 144b171..8d0389a 100644
--- a/sound/soc/soc-core.c
+++ b/sound/soc/soc-core.c
@@ -20,8 +20,6 @@
  *   o Support TDM on PCM and I2S
  */
 
-#define DEBUG
-
 #include <linux/module.h>
 #include <linux/moduleparam.h>
 #include <linux/init.h>
